src/se/vidstige/jadb/DeviceWatcher.java

```diff
@@ -0,0 +1,44 @@
+package se.vidstige.jadb;
+
+import java.util.List;
+import java.io.IOException;
+
+public class DeviceWatcher implements Runnable {
+    private Transport transport;
```

Comment:
No, I saw that later, that's why I wrote "nvm" as in nevermind. :-) Don't worry about it, it was a misstake on my part, this is great. :+1: